sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/options/BlobParallelUploadOptions.java

```diff
@@ -97,4 +133,17 @@ public BlobParallelUploadOptions setRequestConditions(BlobRequestConditions requ
         this.requestConditions = requestConditions;
         return this;
     }
+
+    public Duration getTimeout() {
+        return this.timeout;
+    }
+
+    /**
+     * @param timeout An optional timeout value beyond which a {@link RuntimeException} will be raised.
```

Comment:
Also, document that this will be ignored for async operations.

Reply:
I will add that in the stg73 branch when I update it there. Right now this only applies to sync, so it won't make sense to mention that.